lte/cloud/configs/subscriberdb.yml

```diff
@@ -12,6 +12,9 @@
 
 # flatDigestEnabled is a feature flag for the flat digest functionality.
 flatDigestEnabled: true
```

Comment:
While we're here: let's change this to "digestsEnabled" since this guards more than just the flat digests